key/resolver_test.go

```diff
@@ -101,6 +101,7 @@ func TestMultiResolver(t *testing.T) {
 	loader, err := expander.Expand(
 		map[string]interface{}{KeyIdParameterName: keyId},
 	)
+	assert.Nil(err)
```

Comment:
`assert.NoError` is a better, self-documenting way of doing the same thing as `assert.Nil`, but for errors